src/Cake.Common/Solution/Project/ProjectProjectReference.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Cake.Core.IO;
+
+namespace Cake.Common.Solution.Project
+{
+    /// <summary>
+    /// Represents a project reference to another project.
+    /// </summary>
+    /// <remarks>
+    /// Schema from https://msdn.microsoft.com/en-us/library/ms164283.aspx
+    /// and https://msdn.microsoft.com/en-us/library/bb629388.aspx
+    /// </remarks>
+    public sealed class ProjectProjectReference
```

Comment:
If the other one is renamed to `ProjectAssemblyReference`, then maybe this one can be renamed to simply `ProjectReference`?